tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/AspNetCore/AspNetCoreMvc31TraceId128BitTests.cs

```diff
@@ -0,0 +1,94 @@
+// <copyright file="AspNetCoreMvc31TraceId128BitTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#if NETCOREAPP3_1
+#pragma warning disable SA1402 // File may only contain a single class
+#pragma warning disable SA1649 // File name must match first type name
+using System;
+using System.Net;
+using System.Threading.Tasks;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.TestHelpers;
+using VerifyXunit;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests.AspNetCore
+{
+    public class AspNetCoreMvc31TraceId128Bit : AspNetCoreMvc31TraceId128BitTests
+    {
+        public AspNetCoreMvc31TraceId128Bit(AspNetCoreTestFixture fixture, ITestOutputHelper output)
+            : base(fixture, output, enableQueryStringReporting: false)
```

Comment:
I think there's a typo, it doesn't look like this subclass or the 64Bit subclass are setting the `generate128BitTraceIds` parameter in the base class's constructor.

Reply:
Sorry, the snapshot tests are still WIP. I should've noted that so you didn't waste your time with them.

Reply:
Should be fixed now.